src/components/Notification/Notification.js

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+const Notification = ({ text, status }) => (status ? (
```

Comment:
Notification will be beefed up in the future to be less ambiguous and more featureful